test/test_docstrings.py

```diff
@@ -26,6 +27,59 @@ def test_rst(s):
     assert not s or ret
 
 
+@given(text())
+def test_rst_extract_options_fuzz(s):
+    assume(not s.startswith(":"))
+    content, options = docstrings._rst_extract_options(s)
+    assert not options
+    assert content == s
```

Comment:
The assertions here are overly restrictive I think. For example, you could have ` :foo:bar` as input, which would correctly parse as an option.

IMO the best approach here is to just skip the assertions, the test then only ensures that we don't unexpectedly raise anything.